build/cumulusci.xml

```diff
@@ -1051,6 +1051,9 @@ project.setProperty("downgradePackage", result);
               <delete dir="@{stagedir}" />
               <copy todir="@{stagedir}">
                 <fileset dir="@{subdir}"/>
+                <filtermapper>
+                  <replacestring from="___NAMESPACE___" to="@{nsprefix}"/>
```

Comment:
I chose to use underscores instead of percents because a quick search labelled it as a symbol to not use in filenames.